examples/java/src/test/java/dev/selenium/browsers/ChromeTest.java

```diff
@@ -25,5 +25,10 @@ public void headlessOptions() {
         options.addArguments("--headless=new");
         driver = new ChromeDriver(options);
     }
+    @Test
+    public void browserStartInspecifiedLocation() {
+        ChromeOptions options = new ChromeOptions().setBinary("Path to chrome binary");
```

Comment:
Example tests runs in CI , may be this need to be skipped

Reply:
ok

Reply:
@harsha509 do you think we could set `CHROMIUM_BIN=$(which chrome)` in the github workflow and then reference things by environment variable in our docs?